amgut/db/patches/0041.sql

```diff
@@ -292,6 +517,280 @@ INSERT INTO pm.master_mix_lot (name)
 INSERT INTO pm.water_lot (name)
     VALUES ('RNBD9959');
 
+-- Add the barcode sequence plates
+INSERT INTO pm.barcode_sequence_plate (name, plate_type_id)
+    VALUES ('Primer plate 1', 1), ('Primer plate 2', 1), ('Primer plate 3', 1),
+           ('Primer plate 4', 1), ('Primer plate 5', 1), ('Primer plate 6', 1),
+           ('Primer plate 7', 1), ('Primer plate 8', 1);
+
+INSERT INTO pm.barcode_sequence_plate_layout (
+        barcode_sequence_plate_id, row, col, linker_primer_sequence, barcode_sequence)
+    VALUES
+        -- Primer plate 1
+        (1, 0, 0, 'GTGTGCCAGCMGCCGCGGTAA', 'AGCCTTCGTCGC'), (1, 0, 1,  'GTGTGCCAGCMGCCGCGGTAA', 'TCCATACCGGAA'), (1, 0, 2,  'GTGTGCCAGCMGCCGCGGTAA', 'AGCCCTGCTACA'),
```

Comment:
I think this is a good point. There should be a field that describes what region (e.g., 16S V4, PMID: ####) it is targetting.